src/Microsoft.AspNet.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Hosting.Server;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class WebHostBuilderExtensions
```

Comment:
Basic? Core? Why? This is the original library, everyone else can pick a worse name :smile: